src/frontend/web/src/ratingpage/RatingCard.css

```diff
@@ -0,0 +1,66 @@
+.rating-card {
```

Comment:
Why didn't you use ant's styling?
Also here - please change the suffix to `scss`.

Reply:
By styling you mean ant’s components’ style property?

Reply:
Yes, the card already has a style, and you could change the style a bit to be suitable to our needs, but I don't understand why you need to overwrite it completely (or maybe you didn't, and it is just a wrong impression I got from this css class)

Reply:
I didn't overwrite it, it's additional styling :)
I didn't add it to the card's style property because it's too long